tests/plotting/test_property.py

```diff
@@ -3,6 +3,10 @@
 import pyvista as pv
 from pyvista.plotting._property import _check_range
 
+skip_lesser_9_3_X = pytest.mark.skipif(
+    pv.vtk_version_info < (9, 3), reason="Functions not implemented before 9.3.X"
+)
+
```

Comment:
Since there's just one function skipping, should we just define this at the decorator?

Reply:
Let's remove it.
```suggestion
```